startup/gui/menus.py

```diff
@@ -66,28 +66,38 @@
 ## Help menu
 ###########################################################################
 
-for menuItem, url in [
+def addHelpMenuItems( items ) :
```

Comment:
This seems unrelated to the change we're trying to make. Is there any reason why the one-line diff on L91 isn't sufficient here?

Reply:
It's to allow us to insert the Examples sub menu half way down, without repeating the 'for loop' that does the actual insertion. Just trying to be DRY.

Reply:
Sorry, I missed that somehow - ignore me.